README.md

```diff
@@ -36,6 +36,13 @@ This is a huge benefit for our designers, who can now just edit a textbox or upl
 yarn install
 ```
 
+## Set up Contentful content model
+This project comes with a Contentful setup command `yarn setup`.
```

Comment:
To avoid confusion, I'd rename the `setup` script to `seed-contentful`. Also add a note here in the README to clarify that this script should only be used by contributors without access to the Bits of Good Contentful API keys and that they will need to use their own Contentful space.

Reply:
sure i will add and change it